test/bash/util.bats

```diff
@@ -0,0 +1,158 @@
+
+setup() {
+  load '../../node_modules/bats-support/load'
+  load '../../node_modules/bats-assert/load'
+
+  TEST_DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
+  TEST_ROOT_DIR=$(mktemp -u)
+
+  # Not used yet, but will be later
+  load "$TEST_DIR/../../src/usr/local/buildpack/util.sh"
+}
+
+teardown() {
+  rm -rf "${TEST_ROOT_DIR}"
+}
+
+@test "require_tool" {
+  run require_tool
+  assert_output --partial "No tool defined"
+  assert_failure
+
+  # TODO(Chumper): In my opinion this test should fail
+  run require_tool foo
+  # assert_output --partial "No version defined"
+  # assert_failure
+  assert_success
```

Comment:
@viceice I would like your opinion on this. 
Is this working as intended?

Reply:
i think this should fail